common/src/main/java/dev/ftb/mods/ftbchunks/client/map/MapDimension.java

```diff
@@ -198,13 +147,13 @@ public void release() {
 		}
 
 		regions = null;
-		waypoints = null;
+		waypointManager = null;
 	}
 
 	@Override
 	public void runMapTask() throws Exception {
-		List<Waypoint> waypoints = new ArrayList<>(getWaypoints());
-		List<MapRegion> regionList = new ArrayList<>(getRegions().values());
+		List<Waypoint> waypoints = ImmutableList.copyOf(getWaypointManager());
+		List<MapRegion> regionList = ImmutableList.copyOf(getRegions().values());
```

Comment:
Mainly a code style question, but I feel like this should be addressed nonetheless: Now that Java 9+ has immutable lists as part of its stdlib `List.of / List.copyOf`, should we still be using Guava ImmutableList in our mods, nonetheless?

Reply:
Yeah, we should be moving to `List.of`, you're right.  I'll make a note to do a pass and replace any such instances for the next release.